xlsx_streaming/render.py

```diff
@@ -50,7 +51,10 @@ def render_worksheet(rows_batches, openxml_sheet_string, encoding='utf-8'):
         current_line += 1
     for rows in rows_batches:
         yield render_rows(rows, row_template, start_line=current_line, encoding=encoding)
-        current_line += len(rows)
+        if isinstance(rows, collections.abc.Iterator):
+            current_line += sum(1 for _elem in rows)
```

Comment:
Est-ce que l'appel à `render_rows()` ne va pas consommer l'itérateur (et donc il n'y aura plus d'éléments sur lequels itérer pour calculer la longueur de `rows`) ?

Reply:
Gnark, j'ai cliqué trop vite, je suis d'accord avec toi Damien. Comme solution, soit il faut modifier `render_rows` soit il faut ignorer ce compteur (je n'ai pas tellement regardé à quoi il sert)...